plugins/modules/firewalld.py

```diff
@@ -992,6 +1001,13 @@ def main():
         changed, transaction_msgs = transaction.run()
         msgs = msgs + transaction_msgs
 
+        # Type of masquerade will be changed to boolean in a future release.
+        try:
+            boolean(masquerade, True)
+        except TypeError:
+            warnings.append('The value of masquerade is "%s", It should be set boolean '
+                            'and will be changed from str to bool in a future release.' % masquerade)
```

Comment:
```suggestion
            module.warn('The value of the masquerade option is "%s". '
                        'The type of the option will be changed from string to boolean in a future release. '
                        'To avoid unexpected behavior, please change the value to boolean.' % masquerade)
```

Reply:
done!